### PR TITLE
Add a column to hex.search output showing stable version

### PR DESCRIPTION
**Pull request by dotboris**
Fixes https://github.com/hexpm/hexpm/issues/571

New output (notice that I changed the `Version` column to `Latest` to avoid confusion

```
$ mix run -e 'Mix.Tasks.Hex.Search.run(["postgrex"])'
Package       Description                                            Latest      Stable  URL
postgrex      description                                            1.0.0-rc.1  0.13.3  https://hex.pm/packages/postgrex
ecto_network  Ecto types to support MACADDR and Network extensio...  0.6.0       0.6.0   https://hex.pm/packages/ecto_network
geo_postgis   PostGIS extension for Postgrex.                        1.0.0       1.0.0   https://hex.pm/packages/geo_postgis
posterize     erlang wrapper for the postgrex postgres client        0.13.3      0.13.3  https://hex.pm/packages/posterize
hstore        A collection of encoders and decoders for hstore d...  0.0.2       0.0.2   https://hex.pm/packages/hstore
postgrex_cdb  PostgreSQL driver for Elixir compatible with Cockr...  0.13.3      0.13.3  https://hex.pm/packages/postgrex_cdb
seek          A simple wrapper around Postgrex to simplify writi...  0.0.1       0.0.1   https://hex.pm/packages/seek
```

### CODE REVIEW

File: lib/mix/tasks/hex.search.ex [hunk 1]
```diff
@@ -39,16 +39,39 @@ defmodule Mix.Tasks.Hex.Search do
...
+          Hex.Utils.truncate(package["meta"]["description"] |> trim_heredoc),
+          latest(package["releases"]),
+          latest_stable(package["releases"]),
+          package["html_url"] || package["url"]
+        ]
       end)
 
-    Mix.Tasks.Hex.print_table(["Package", "Description", "Version", "URL"], values)
+    Mix.Tasks.Hex.print_table(
+      ["Package", "Description", "Latest", "Stable", "URL"],
```
Comment (ericmj) [inline lib/mix/tasks/hex.search.ex]: We should only show the latest stable version. No need to separate latest and stable.

File: lib/mix/tasks/hex.search.ex [hunk 2]
```diff
@@ -39,16 +39,39 @@ defmodule Mix.Tasks.Hex.Search do
...
+    )
   end
 
   defp latest([%{"version" => version} | _]) do
     version
   end
 
+  defp latest_stable(releases) do
+    %{"version" => version} = releases
+    |> Enum.find(%{"version" => nil}, &is_stable/1)
```
Comment (ericmj) [inline lib/mix/tasks/hex.search.ex]: Avoid single pipes.

File: lib/mix/tasks/hex.search.ex [hunk 3]
```diff
@@ -39,16 +39,39 @@ defmodule Mix.Tasks.Hex.Search do
...
+
+    version
+  end
+
+  defp is_stable(%{"version" => version}) do
+    case Version.parse(version) do
```
Comment (ericmj) [inline lib/mix/tasks/hex.search.ex]: Use `Hex.Version.parse!/1` instead.

**Comment by dotboris** [comment]
@ericmj Looks like the unit tests for `hex.outdated` timed out for some reason. Sounds like the test could be flimsy. Could you try running the travis job again? I can't do it myself.

**Comment by ericmj** [comment]
Sure, restarted.

**Comment by dotboris** [comment]
All is green in tarvis land 👌

**Comment by dotboris** [comment]
@ericmj sorry to pester you. Is there anything left for this to get merged?

**Comment by ericmj** [comment]
@dotboris It needs one more review.

/cc @wojtekmach @josevalim

**Comment by wojtekmach** [comment]
Thank you!

**Comment by ericmj** [comment]
Thank you @dotboris!

**Pull request merged**